src/components/color-system/examples/extended-color-palette-example.tsx

```diff
@@ -0,0 +1,168 @@
+                    <div class="swatch --color-amber-darker"></div>
+                    <div class="swatch hue">amber</div>
+                    <div class="swatch --color-orange-lighter"></div>
+                    <div class="swatch --color-orange-light"></div>
+                    <div class="swatch --color-orange-default">·</div>
+                    <div class="swatch --color-orange-dark"></div>
+                    <div class="swatch --color-orange-darker"></div>
+                    <div class="swatch hue">orange</div>
+                    <div class="swatch --color-coral-lighter"></div>
+                    <div class="swatch --color-coral-light"></div>
+                    <div class="swatch --color-coral-default">·</div>
+                    <div class="swatch --color-coral-dark"></div>
+                    <div class="swatch --color-coral-darker"></div>
+                    <div class="swatch hue">coral</div>
+                    <div class="swatch --color-brown-lighter"></div>
+                    <div class="swatch --color-brown-light"></div>
+                    <div class="swatch --color-brown-default">·</div>
+                    <div class="swatch --color-brown-dark"></div>
+                    <div class="swatch --color-brown-darker"></div>
+                    <div class="swatch hue">brown</div>
+                    <div class="swatch --color-gray-lighter"></div>
+                    <div class="swatch --color-gray-light"></div>
+                    <div class="swatch --color-gray-default">·</div>
+                    <div class="swatch --color-gray-dark"></div>
+                    <div class="swatch --color-gray-darker"></div>
+                    <div class="swatch hue">gray</div>
+                    <div class="swatch --color-glaucous-lighter has-brightness-label lighter"></div>
+                    <div class="swatch --color-glaucous-light has-brightness-label light"></div>
+                    <div class="swatch --color-glaucous-default has-brightness-label default">
+                        ·
+                    </div>
+                    <div class="swatch --color-glaucous-dark has-brightness-label dark"></div>
+                    <div class="swatch --color-glaucous-darker has-brightness-label darker"></div>
+                    <div class="swatch hue">glaucous</div>
+                </div>
+                <limel-flex-container justify="start">
+                    <limel-checkbox
+                        label="Highlight Lime's brand colors"
+                        onChange={this.toggleMode}
+                        checked={this.brandColors}
+                    />
+                </limel-flex-container>
+                <div class="brand-colors-tips">
+                    Most of the brand colors are included in the subset for "
+                    <b>dark mode</b>". Switch between dark & light mode to see
+                    all of them.
+                </div>
```

Comment:
I'm thinking we should maybe put these examples onto a background contrast colour, so that the reader, 1) is reminded what their current setting is (I keep forgetting I'm in dark mode all the time, because the background is white), and, 2) sees the colour against the intended background?

Reply:
good idea!

Reply:
:zap: